### PR TITLE
Updates to risk block and codelists

### PR DESCRIPTION
**Pull request by duncandewhurst**
Make riskCategory an open codelist and update codes and definitions in
line with consultation feedback. Remove likelihood and fiscalImpact
properties. Remove shared code from riskAllocation codelist and replace
with 'undefined'

### CODE REVIEW

**Review by timgdavies — CHANGES_REQUESTED** [review]
To fit with 1.1 we need to:

(1) Remove the merge strategies;

(2) Include 

``` "codelist":"codelist.csv", 
      "openCodelist":false 
```
properties against each field with codes, and remove the enums.

**Comment by duncandewhurst** [comment]
@timgdavies I've made those updates in [452b123](https://github.com/open-contracting/ocds-riskAllocation-extension/pull/2/commits/452b123742cd72ce2f4ef7d1218fb45f7e7c4b3f) and also update the riskCategory codelist in line with Abdou's latest email.

Ok to merge now?

File: release-schema.json [hunk 1]
```diff
@@ -11,40 +11,23 @@
...
-            "forcemajeure",
-            "governmentAction",
-            "changeInLaw",
-            "rebalancing",
-            "renegotiation",
-            "contractTermination",
-            null
-          ],
-          "mergeStrategy": "ocdsVersion"
+          ]
```
Comment (timgdavies) [inline release-schema.json]: add

```json
"codelist":"riskCategory.csv",
"openCodelist":false
```

File: release-schema.json [hunk 2]
```diff
@@ -53,61 +36,23 @@
...
-            "null"
-          ],
-          "enum": [
-            "low",
-            "medium",
-            "high",
-            null
-          ],
-          "mergeStrategy": "ocdsVersion"
+		"openCodelist": "false"
```
Comment (timgdavies) [inline release-schema.json]: Add

```json
"codelist":"riskAllocation.csv",
"openCodelist":false
```

(note no quotes around false)

**Review by timgdavies — CHANGES_REQUESTED** [review]
Small tweaks still needed to schema I'm afraid. Check the 1.1 scheme for the syntax for codelist markup etc. http://standard.open-contracting.org/1.1-dev/en/release-schema.json

**Comment by duncandewhurst** [comment]
@timgdavies thanks - updated inline with your comments, except for riskCategory which is an open codelist so have set openCodelist = true

**Review by timgdavies — APPROVED** [review]

**Pull request merged**